anyway/backend_constants.py

```diff
@@ -120,3 +85,111 @@ class ResolutionCategories(Enum):
 
 
 BE_CONST = BackEndConstants()
+
+
+class LabeledCode(Enum):
+    @staticmethod
+    def get_label_impl(cls, val: int, labels: List[str]) -> str:
+        if 0 <= val - 1 < len(labels):
+            return labels[val - 1]
+        else:
+            err = f'{val}: not found in class {cls}'
+            logging.error(err)
+            raise ValueError(err)
+
+    @classmethod
+    def codes(cls: Iterable) -> List[int]:
+        if isinstance(cls, collections.Iterable):
+            # pylint: disable=E1133
+            return [a.value for a in cls]
+        else:
+            raise ValueError(f'{cls}: needs to be derived from Enum')
+
+    # def get_label(self) -> str:
+    #     return self.labels[self.value]
+
+
+# This is a type for the field 'injury_severity' in the table 'involved_markers_hebrew'
+class InjurySeverity(LabeledCode):
+    FATAL = 1
+    SEVERE = 2
+    LIGHT = 3
+
+    @classmethod
+    def get_label(cls, val: int) -> str:
+        labels = ["fatal", "severe", "light"]
+        return LabeledCode.get_label_impl(cls, val, labels)
```

Comment:
If `get_label_impl()` would be a `classmethod`, this call becomes `cls.get_label_impl(val, labels)` which is somewhat preferable since it does not repeat the name of the class.

If the implementation of `get_list_of_names()` would return the list of names and nothing else, there would be no need to write this call at all, and `LabeledCode.get_label()` would simply query this method:
```python
    @classmethod
    def get_label(cls, val: int) -> str:
        labels = cls.get_labels()
        if 0 <= val - 1 < len(labels):
            return labels[val - 1]
        else:
            err = f'{val}: not found in class {cls}'
            logging.error(err)
            raise ValueError(err)
```